src/main/java/teammates/ui/template/FeedbackSubmissionEditQuestion.java

```diff
@@ -35,6 +38,9 @@ public FeedbackSubmissionEditQuestion(FeedbackQuestionAttributes questionAttribu
         isRecipientNameHidden = questionAttributes.isRecipientNameHidden();
         isGiverTeam = questionAttributes.giverType.equals(FeedbackParticipantType.TEAMS);
         isRecipientTeam = questionAttributes.recipientType.isTeam();
+        isRecipientOwnTeamMembers = questionAttributes.recipientType.isOwnTeamMembers();
+        isRecipientOtherTeams = questionAttributes.recipientType.isOtherTeams();
```

Comment:
The first approach won't work. But we can add just one variable, instead of 3 booleans i,e `messageToDisplayOnNoRecipientAvailable`. We will check here itself which message to display and use this in jsp?

Or second approach is also fine.